example-config.yml

```diff
@@ -332,6 +332,27 @@ itinerary:
 #       - WALK
 #       - BICYCLE
 
+### Configure here the carbon output in grams per meter for each mode.
+### massUnits can be 'pound', 'kilogram', 'ounce', or 'gram'
```

Comment:
I would move all the carbonIntensity values to core-utils and then make massUnit the only valid setting and then we can use typescript to enforce these options

Reply:
You don't think the agencies might want to customize the carbon intensity of each mode?